pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = [
+    "setuptools",
+    "wheel",
+    "Cython>=0.28.5",
+    "numpy>=1.11.0",
+    "scipy>=0.17.0",
```

Comment:
#15106 will be merged soon. Let's bump-up numpy to 1.13.3 and scipy to 0.19.1.

Reply:
merged :)